unison-share-api/package.yaml

```diff
@@ -9,14 +9,14 @@ library:
       other-modules: Paths_unison_share_api
 
 dependencies:
-  - NanoID
```

Comment:
This ended up bitrotting, but we only used it for generating UI tokens, so I just rewrote that bit in terms of Crypto.Random